metrics/web/docs/features/remote_config/diagrams/remote_config_presentation_class.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+
+package base.domain.usecases {
+    class GetRemoteConfigUseCase{}
+}
+
+package common.presentation.state {
+    class RemoteConfigNotifier {
+        + isLoginFormEnabled : bool
+        + isFpsMonitorEnabled : bool
+        + isRendererDisplayEnabled : bool
+        + isLoading : bool
+
+        - _remoteConfig : RemoteConfig
+        - _getRemoteConfigUseCase : GetRemoteConfigUseCase
+        - Future<void> _initializeRemoteConfig()
```

Comment:
What about the public class that should trigger initialization? Seems like we shouldn't trigger it within the constructor.